src/ng-package.schema.json

```diff
@@ -54,6 +54,7 @@
         "entryFile": {
           "description": "Entry file to the public API (default: `src/public_api.ts`).",
           "type": "string",
+          "pattern": "^(?!index\\.ts).*$",
```

Comment:
I have one question: If people cannot have `index.ts` in the sources, does that cause any problems with path mappings? e.g. assume I have a library in my CLI project and an application. The application will depend on my local library and somewhere I have a `tsonfig.json` with `paths` mappings. e.g.

```json
"paths": [
  "my-lib": ["libs/my-lib/public_api.ts"]
]

Reply:
This is actually not recommended, in the Angular CLI that path mappings will always point to the built library.

Reply:
<img width="810" alt="Screenshot 2022-05-05 at 11 59 28" src="https://user-images.githubusercontent.com/17563226/166901589-ef448e4f-0217-4bb4-b5bc-47a89d0f1a3c.png">

https://angular.io/guide/creating-libraries

Reply:
I see, that makes sense. sounds like we are good then!